index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Figma Template 1</title>
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
+    <link rel="stylesheet" href="style.css">   
+</head>
+<body>
+    <header>
+        <div class="logo"><a>Leon Espitia</a></div>
+        <nav class="menu_icon">
+            <div>
+              <img src="./assets/icons/Hamburguer.svg" alt=""/>
+            </div>  
+        </nav>
+    </header>
+
+    <main>
+        <section class="headline">
+          <h1 class="title">I'm Leon. Happy to see you!</h1>
+          <p class="headline_content">
+            I'm a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.
+          </p>
+
+          <p class="lets">LET'S CONNECT</p>
+          
+          <div class="social_icons">
+              <a href="" target="_blank">
+                  <img src="./assets/icons/Twiter.svg" alt=""/>
+              </a>
+              <a href="" target="_blank">
+                <img src="./assets/icons/Linkedin_icon.png" alt=""/>
+              </a>
+              <a href="" target="_blank">
+                <img src="./assets/icons/M.svg" alt=""/>
+              </a>
+              <a href="" target="_blank">
+                <img src="./assets/icons/Github.svg" alt=""/>
+              </a>
+              <a href="" target="_blank">
+                <img src="./assets/icons/Fin.svg" alt=""/>
+              </a>
+          </div>
```

Comment:
- [ ] Please make sure that The list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has `<a>` link tag. for example:
```
<ul>
    <li>
        <a href="" target="_blank">
             <img src="./assets/icons/Twiter.svg" alt=""/>
        </a>
    </li>
    .................
    .................
</ul>
```

Reply:
Hi! Omar-Muhamad
I really appreciate all your comments and suggestions.
Thank you.